external/netstandard/netstandard1.x/project.json.template

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "NETStandard.Library": "2.0.0-beta-24919-01",
+        "System.Diagnostics.Contracts": "4.3.0"
```

Comment:
I don't like it because that means a vertical build will download packages from NuGet.  I separated it so that we wouldn't download anything we didn't build in another repo for a vertical build.